src/usb/noise.c

```diff
@@ -179,7 +179,9 @@ static bool _process_handshake(
         // We are paired now, so we pop that screen.
         // Pairing is the start of a session, so we clean the screen stack in case
         // we started a new session in the middle of something.
+#if PLATFORM_BITBOX02 == 1
```

Comment:
if you can remember at all why this was needed, it would be great to put into the commit message or in the comment. it will be hard to reason about later.

Reply:
The issue is that in the bitboxbase we always have a screen running at the root of the stack.